build/node-compose.yml

```diff
@@ -29,11 +27,12 @@ services:
       CARTESI_BLOCKCHAIN_IS_LEGACY: "false"
       CARTESI_BLOCKCHAIN_GENESIS_BLOCK: "1"
       CARTESI_BLOCKCHAIN_FINALITY_OFFSET: "1"
-      CARTESI_CONTRACTS_DAPP_ADDRESS: "0x70ac08179605AF2D9e75782b8DEcDD3c22aA4D0C"
+      CARTESI_BLOCKCHAIN_READ_DEPTH: "1"
+      CARTESI_CONTRACTS_DAPP_ADDRESS: "0x180763470853cAF642Df79a908F9282c61692A45"
       CARTESI_CONTRACTS_DAPP_DEPLOYMENT_BLOCK_NUMBER: "1"
-      CARTESI_CONTRACTS_HISTORY_ADDRESS: "0x4FF8BD9122b7D91d56Dd5c88FE6891Fb3c0b5281"
-      CARTESI_CONTRACTS_AUTHORITY_ADDRESS: "0x5050F233F2312B1636eb7CF6c7876D9cC6ac4785"
-      CARTESI_CONTRACTS_INPUT_BOX_ADDRESS: "0x59b22D57D4f067708AB0c00552767405926dc768"
+      CARTESI_CONTRACTS_HISTORY_ADDRESS: "0xbCc4067dfdeE8274E531951f0968EC895f397A44"
+      CARTESI_CONTRACTS_AUTHORITY_ADDRESS: "0xb090149a3cA43000681B74875500854B54355496"
+      CARTESI_CONTRACTS_INPUT_BOX_ADDRESS: "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9"
```

Comment:
why doesn't it have the same addresses of the testnets and mainnets?

Reply:
See #229